packages/client-sdk-nodejs/src/internal/internal-auth-client.ts

```diff
@@ -71,10 +71,10 @@ export class InternalAuthClient implements IAuthClient {
     ];
   }
 
-  public async generateAuthToken(
+  public async generateApiKey(
     scope: TokenScope,
```

Comment:
What about `TokenScope`? Are we still saying that an ApiKey is a [type of] Token?

Reply:
I can create an alias for this if we think it's worthwhile. @allenheltondev wdyt

Reply:
In https://github.com/momentohq/client-sdk-javascript/pull/838/commits/fb9ba8d38844cc18a8cbc34656c71552c0cb8716 I changed the name of this to `PermissionScope`. Can revert that commit if we decide to keep the old name.